CppCoreGuidelines.md

```diff
@@ -12376,18 +12378,20 @@ Aternatively, we will decide that no change is needed and delete the entry.
 * What to do with leaks out of temporaries? : `p = (s1+s2).c_str();`
 * pointer/iterator invalidation leading to dangling pointers
 
-	void bad()
-	{
-		int* p = new int[700];
-		    int* q = &p[7];
-    	delete p;
+```
```

Comment:
For now, we are trying to avoid fencing.  Could you rewrite this part of the patch with indentation only?
Thanks!